### ISSUE TITLE
Support writing to xml

### ISSUE THREAD

**Issue by cdilga** [mindset/scorm#4]
So far as I can tell there is no support for changing any of the identifiers and writing these to a manifest file. Is there any support for writing changes to imsmanifests?

**Comment by niho** [comment]
You are correct. The library is currently not designed to programmatically edit SCORM packages or imsmanifest files. It does have the `create`, `bundle` and `check` command that can be used to create SCORM packages. But you will need to edit the imsmanifest file manually.
